pyproject.toml

```diff
@@ -86,7 +86,7 @@ line-length = 88
 target-version = 'py310'
 
 [tool.ruff.lint.per-file-ignores]
-"__init__.py" = ['I', 'F403']
+"__init__.py" = ['I', 'F403', 'F401']
```

Comment:
`'F401'` is added here to not get "module imported but unused." while running pre-commit on the `nx_parallel/__init__.py` because we just import `_configure_if_nx_active` in the file to include it in the public API but we don't use it.